src/tools/devtools/memory.md

```diff
@@ -46,6 +46,17 @@ is no longer used (see [Dart garbage collection][]).
 
 [Dart garbage collection]: {{site.medium}}/flutter/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30
 
+### Object types
+
+#### Disposable object
+
+A Dart object, that has method `dispose()`, which, by de-facto convention, is expected to be invoked,
+when the object is not needed any more, to avoid memory leaks.
```

Comment:
wording/ punctuation nit: "A Dart object that has a method `dispose()`, which, by convention, is expected to be invoked when the object is not needed anymore to avoid memory leaks."